### ISSUE TITLE
New location: Quarry

### ISSUE THREAD

**Issue by Zarhon** [CleverRaven/Cataclysm-DDA#74955]
### Is your feature request related to a problem? Please describe.

With the removal of craters, survivors now have less options to acquire a large amount of rocks or minerals (whether for forging, chemical or construction projects), outside of looting mines, manually digging or demolishing houses. 

At the same time, the mines in their current state are very repetitive, and always contain some eldritch horror at the bottom, or are plain annoying to traverse for most survivors.

Quarries would solve this problem and add to the realism of the world.

### Solution you would like.

Quarries would be locations similar to a mine, but much bigger tiles-wise (somewhere along the size of the 'big crater' from before) and rare enough to not pop up all over the place (e.g. about as common as a mutant beehive. 

It would have miner zombies around as its main threat, and their evolutions. It could either be a simple pit (like the slime pit) full of piles of dirt, rock, minerals and construction gear, or be a circular downwards ramp, as most cliff-side quarries are done. 

Purpose of quarries: Supplying survivors with miner zombie gear, massive amounts of rocks or other minerals (like a mine could, but much, much bigger quantities). At the outer edges of the quarry, survivors can find shacks where the miners worked/relaxed, construction trucks or 'dozers' designed to carry/dig/move large amounts of rock, rubble, sand and such, and possibly large construction machines (which wouldn't be vehicles so much as buildings) which offer height vision (for construction cranes), or potent pieces of gear to strip for high-end vehicle building. The place would also naturally offer a mix of mine and construciton site tools and loot. Rarely, they might have something like an artifact at the bottom (which was unearthed by accident and caused something bad), or simply a ton of explosives stashed nearby, the kind used for such tunneling projects.

EDIT: Apparently quarries would get flooded. This would make them potentially interesting locations to explore with scuba gear.

### Describe alternatives you have considered.

Not doing this.

### Additional context

Some examples of quarries or machinery
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/39234303/758abac1-7d07-4893-89fc-7d2c2f715663)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/39234303/5e6919bb-7294-42ef-8506-c0a2d1395802)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/39234303/25fe4c4d-ad88-410a-935a-a40754a1d077)

**Comment by kevingranade** [comment]
No problem with having quarries in general, but they would not have the "purpose" of supplying raw materials to the survivor,  those are much easier to come by elsewhere and especially what you really want is processed materials like slabs or bricks or even concrete, not huge unwieldy chunks of raw rock.
Also keep in mind *every* quarry is now abandoned, which tends to result in the pit filing with water, especially since it wasn't closed cleanly but simply abandoned.
Finally,  the materials retrieved are going to be dominated by gravel,  sand,  and bulk stone,  none of which are particularly useful.

**Comment by zachary-kaelan** [comment]
I agree with Kevin that quarries should have at least a significant chance of having a lot of water in them. Requiring scuba gear to properly explore quarries makes them cooler on principle. You'll just have to figure out what worthwhile things the player will find down there. Precious materials will probably have not just been laying around in a quarry so it'll have to be something out of the box. Maybe something troglodyte mutagen related.

**Comment by Consoleable** [comment]
Maybe have a couple of quarrys, and one or two versions holds a lab entrance? Quarries would be potentially ideal for hiding a secret lab, one of the difficult things about secret constructions is what to do with all the filldirt/stone produced by excavation, and a quarry would be ideal for this. Plus it allows large vehicles to discreetly transport supplies to and from the site under false cover of minerals and semi-trucks. 
Alternatively, a secret LMOE shelter from some affluent who bought a failing quarry and wanted a discreet lmoe.
A flooded lmoe/lab could be a fun water dungeon to use some of the scuba stuff we have in the game. There might be some vertcality allowing for non-flooded portions of the aforementioned buildings above the flooded entrace/layers. Since the player would have to descend the quarry strip first, you could have multiple z-levels of these dungeons without needing a hill.
(player descends 3 levels down the quarry strip, goes in entrance, now you have room for <=2 floors above the bottom of the quarry)

**Comment by Zarhon** [comment]
I like the idea of them potentially being flooded (wouldn't that take ages to happen, though?) and/or a hidden lab (maybe a lab still being built?) and being an incentive to explore with scuba gear - and possibly add some 'inland' aquatic monsters or even aquatic mutants, like horrifying fish-people that escaped from a now-flooded aquatic lab, or the Blob producing an underwater version of the fallen tower.

**Comment by zachary-kaelan** [comment]
> horrifying fish-people that escaped from a now-flooded aquatic lab

You mean [Deep Ones](https://lovecraft.fandom.com/wiki/Deep_One)?
